man/multicharge.1.adoc

```diff
@@ -20,7 +20,7 @@ Hint for the format of the input file
 *--json*::
 Provide output in JSON format to the file 'multicharge.json'
 
-*--grad*::
+*-g, --grad*::
```

Comment:
```suggestion
*-g, -grad, --grad*::
```

Reply:
I also just realized that the existing flags are not consistent either. 😄